cardano-cli/test/cardano-cli-golden/Test/Golden/CreateTestnetData.hs

```diff
@@ -54,6 +54,8 @@ hprop_golden_create_testnet_data =
 
     H.diffVsGoldenFile generated'' "test/cardano-cli-golden/files/golden/conway/create-testnet-data.out"
 
+-- | This test tests the non-transient case, i.e. it generates strictly
```

Comment:
Did you mean it tests the transient case?

Reply:
Indeed, good catch :+1: Fixed :+1: